web-client/src/presenter/actions/TrialSessionWorkingCopy/updateCalendaredCaseUserNoteAction.ts

```diff
@@ -16,11 +16,11 @@ export const updateCalendaredCaseUserNoteAction = ({
   const caseToUpdate = calendaredCases.find(
     aCase => aCase.docketNumber === userNote.docketNumber,
   );
-  caseToUpdate.notes = userNote;
+  (caseToUpdate as { notes?: unknown }).notes = userNote;
 
   const userNotes = [];
   for (const calendaredCase of calendaredCases) {
-    userNotes.push(calendaredCase.notes);
+    userNotes.push((calendaredCase as { notes?: unknown }).notes);
```

Comment:
So, I get that `RawCase` doesn't have a `notes` prop, but I think this assertion is a little heavy-handed and is actually causing more headache for you in `extractUserNotesFromCalendaredCaseAction` later on. Extending the `RawCase` type would allow you to use a type annotation once rather than lots of assertions everywhere this is touched.